server/etcdserver/backend.go

```diff
@@ -50,6 +50,11 @@ func newBackend(cfg config.ServerConfig) backend.Backend {
 		// permit 10% excess over quota for disarm
 		bcfg.MmapSize = uint64(cfg.QuotaBackendBytes + cfg.QuotaBackendBytes/10)
 	}
+	if cfg.ExperimentalMemoryMlock {
+		cfg.Logger.Info("setting `Mlock` bbolt flag")
+	}
```

Comment:
I don't think we need this log... at most it should be 'debug'.

Reply:
Idea of it was to replace 

https://github.com/etcd-io/etcd/blob/ed4a87d541d7269fe081afcc0db594961cab37c1/server/etcdserver/server.go#L321

but it is also fair to skip it I guess